src/client/opamAdminCommand.ml

```diff
@@ -830,6 +830,39 @@ let check_command cli =
   Term.(const cmd $ global_options cli $ ignore_test_arg $ print_short_arg
         $ installability_arg $ cycles_arg $ obsolete_arg)
 
+let compare_package_versions_command_doc = "Compare 2 package versions"
+let compare_package_versions_command cli =
+  let version_arg n =
+    let doc =
+      Arg.info
+        ~docv:"VERSION"
+        ~doc:"Package version to compare" []
+    in
+    Arg.(required & pos n (Arg.some' OpamArg.package_version) None & doc)
+  in
+  let command = "compare-package-versions" in
+  let doc = compare_package_versions_command_doc in
+  let man = [
+    `S Manpage.s_description;
+    `P "This command compares 2 package versions for quick sanity checks, and prints the result of the comparison to the console. For example:";
+    `I ("For example:", "opam admin compare-package-versions 0.0.9 0.0.10");
+    `I ("outputs:", "-1 (0.0.9 < 0.0.10)");
+    `S Manpage.s_arguments;
+    `S Manpage.s_options;
+  ]
+  in
+  let cmd global_options v1 v2 () =
+    OpamArg.apply_global_options cli global_options;
+    let result = OpamPackage.Version.compare v1 v2 in
+    OpamConsole.formatted_msg "%d (%s %s %s)\n"
+      result
```

Comment:
what is the goal of showing the raw int in the message? Is it to simplify parsing if someone were to use this command in a script? If so `cut '-d ' -f2` would already give that information
```suggestion
    OpamConsole.formatted_msg "%s %s %s\n"
```

Reply:
Hi @kit-ty-kate, and thank you for starting the review already!

This is reminiscent of a distant (and perhaps obsolete) past where some `compare` functions would return non-normalized results, and I wasn't sure whether the internal int would be of interest here.

Your question prompted me to look at `OpamVersionCompare` more closely, and indeed it uses a `normalize_comp_result` step.

Given this, I'm happy to remove the int from the output.

By the way, I appreciate the use of `Ordering`-like modules nowadays and feel enthusiastic about seeing it integrated into more projects over time (perhaps a good candidate for an addition to `OpamStd` later down the road?).

Reply:
Sorry I had accepted your commit suggestion through the GitHub interface as a separate commit but ended up amending it by accident. I guess you'll squash all commits at the end?